### ISSUE TITLE
alpakka-json-streaming cannot be used from Maven as-is

### ISSUE THREAD

**Issue by laszlovandenhoek** [akka/alpakka#2338]
### Versions used 

2.0.0

### Expected Behavior

Adding `akka-stream-alpakka-json-streaming_2.xx` as a Maven dependency works out of the box.

### Actual Behavior

Given a `pom.xml` like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <groupId>test</groupId>
    <artifactId>test</artifactId>
    <version>0.0.1-SNAPSHOT</version>

    <dependencies>
        <dependency>
            <groupId>com.lightbend.akka</groupId>
            <artifactId>akka-stream-alpakka-json-streaming_2.13</artifactId>
            <version>2.0.0</version>
        </dependency>
    </dependencies>
</project>
```
Running `mvn verify` from the command line yields this error:

`[ERROR] Failed to execute goal on project test: Could not resolve dependencies for project test:test:jar:0.0.1-SNAPSHOT: Could not find artifact com.github.jsurfer:jsurfer:jar:1.6.0 in nexus`

The issue is that there __is__ no JAR, only a POM. However, in the published `pom.xml` for `alpakka-json-streaming`, the dependency on `com.github.jsurfer:jsurfer:jar:1.6.0` is not qualified as a `pom` dependency, and thus assumed to be a JAR. It is specified here: 

https://github.com/akka/alpakka/blob/master/project/Dependencies.scala#L301

**Comment by ennru** [comment]
I'm afraid sbt can not express this correctly.

**Comment by ennru** [comment]
Can we depend on the libraries it points to directly?

**Comment by laszlovandenhoek** [comment]
Until the PR is merged and a new version is released, you can work around the problem by excluding the erroneous dependency like so:

```
<dependency>
    <groupId>com.lightbend.akka</groupId>
    <artifactId>akka-stream-alpakka-json-streaming_2.13</artifactId>
    <version>2.0.0</version>
    <exclusions>
        <exclusion>
            <groupId>com.github.jsurfer</groupId>
            <artifactId>jsurfer</artifactId>
        </exclusion>
    </exclusions>
</dependency>
```

**Comment by ennru** [comment]
Fixed with #2344, but I'll leave it open for others to find the temporary fix.

**Comment by ennru** [comment]
Releasing Alpakka 2.0.2 tomorrow.
